py3status/modules/wifi.py

```diff
@@ -73,6 +73,8 @@ class Py3status:
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
+    _ssid = None
+    _max_bitrate = 0
```

Comment:
How to install?
`cd ~/src/py3status`
`git pull #Already up-to-date`
`sudo python setup install`
`sudo python setup build`
`scp wifi.py # to ~/src/py3status/py3status/modules/`
`i3-msg restart`

If all above OK, I think it may had to do something with this linux distro. For starters, I think they don't do `/usr/local/bin` things... and I could add `self.py3.check_command` in DPMS for `xset`... LOL!